### PR TITLE
enable deleting workspaces

### PR DESCRIPTION
**Pull request by panentheos**
Activated via the context menu within a workspace.

### CODE REVIEW

File: src/pages/workspaces/workspace/WorkspaceContainer.js [hunk 1]
```diff
@@ -37,45 +40,130 @@ const navIconProps = {
...
+      div({ style: { flexGrow: 1 } }),
+      h(Interactive, { as: 'div', ...navIconProps }, [icon('copy', { size: 22 })]),
+      h(PopupTrigger, {
+        ref: this.menu,
+        content: contextMenu([
+          { children: 'Share' },
+          { children: 'Publish' },
+          {
+            children: 'Delete',
+            disabled: !workspace || !_.includes(workspace.accessLevel, ['OWNER', 'PROJECT_OWNER']),
```
Comment (dmohs) [inline src/pages/workspaces/workspace/WorkspaceContainer.js:76]: Would it be possible to also include a message on click? Something like, "You must be an owner of this project or the Google project containing it."

**Review by dmohs — APPROVED** [review]
Just deleted one of my test projects, yay! The UX for this has a nice feel.

File: src/pages/workspaces/workspace/WorkspaceContainer.js (hunk 1 above)
Comment (panentheos) [reply src/pages/workspaces/workspace/WorkspaceContainer.js:76]: It's pretty clearly disabled, so people probably won't try to click it, but we could add a tooltip to describe why.

Comment (panentheos) [reply src/pages/workspaces/workspace/WorkspaceContainer.js:76]: (I'll do that in a followon)

**Pull request merged**